### ISSUE TITLE
Cannot determine asset for chunk

### ISSUE THREAD

**Issue by dkrutsko** [waysact/webpack-subresource-integrity#106]
Please forgive me if this was already reported, however, I believe my situation to be different.

> WARNING in webpack-subresource-integrity: Cannot determine asset for chunk 24, computed="style.js", available=style.css. Please report this full error message along...

This happens because I am extracting CSS styles from `style.js` using [`mini-css-extract-plugin`](https://github.com/webpack-contrib/mini-css-extract-plugin) and removing the now empty `style.js` file using [`webpack-fix-style-only-entries`](https://github.com/fqborges/webpack-fix-style-only-entries). Here is a sample Webpack configuration demonstrating the issue.

```js
const path = require('path');
const FixStyle = require('webpack-fix-style-only-entries');
const Extract = require('mini-css-extract-plugin');
const HTML = require('html-webpack-plugin');
const SRI = require('webpack-subresource-integrity');

module.exports =
{
	mode: 'production',
	context: path.join(process.cwd(), 'source/'),

	entry:
	{
		style: [ 'styles/style'  ],
		entry: [ 'scripts/entry' ]
	},

	output:
	{
		crossOriginLoading: 'anonymous',
		chunkFilename: '[name]-[chunkhash].js',
		filename: '[name]-[contenthash].js',
		path: path.resolve(__dirname, 'public')
	},

	plugins:
	[
		new FixStyle(),

		new Extract({ filename: '[name]-[contenthash].css' }),

		new HTML
		({
			filename: 'index.html',
			template: 'index.ejs'
		}),

		new SRI({ hashFuncNames: [ 'sha384' ] })
	],

	module:
	{
		rules:
		[
			{
				test: /\.css$/,
				use: [ Extract.loader, 'css-loader' ]
			}
		]
	}
};
```

I am using `webpack-subresource-integrity@1.3.2`

**Comment by jscheid** [comment]
Thanks for your bug report. Could you please turn this into a full test case by making a self-contained repository? I've tried making a test case based on the above but it doesn't trigger the warning.

I have a hunch the problem is that webpack-fix-style-only-entries doesn't "fully" remove the JS file from the build. I can see in its source that it deletes the asset, but I think that's not enough. It might also have to remove any references to the assets from other places, in particular from the chunk map.

Anyway, if there's something simple this plugin can do to work around it I'm happy to add that, but first I would need a working test case.

**Comment by jscheid** [comment]
I'm going to close this because without being able to reproduce it there's not much I can do. Please feel free to reopen with some more context, ideally a small repository that demonstrates the issue.

**Comment by jasonrm** [comment]
I made a small repo that will reproduce this warning. 
https://github.com/jasonrm/wsi-106

**Comment by jscheid** [comment]
@jasonrm fantastic, thanks!

**Comment by jscheid** [comment]
A fix for this has been released in version 1.3.4. Thanks again for the bug report and the test case.
